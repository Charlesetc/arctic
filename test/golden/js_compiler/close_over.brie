
define main [
  define x 3
  define f [ x ]
  f ()
]
