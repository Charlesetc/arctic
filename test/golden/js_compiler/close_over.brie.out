function fn_2_(x) {
	return x;
}
function main() {
	var x = 3;
	var f = new_closure().fill_in_arguments(x);
	return f.fill_in_arguments().call(fn_2_);
}